eng/scripts/automation/Invoke-CadlDataPlaneGenerateSDKPackage.ps1

```diff
@@ -52,8 +52,37 @@ param (
   [string]$specRoot = "",
   [string]$repo="",
   [string]$commit="",
-  [string]$additionalSubDirectories="" #additional directories needed, separated by semicolon if more than one
+  [string]$additionalSubDirectories="", #additional directories needed, separated by semicolon if more than one
+  [switch]$help
 )
+if ($help) {
+  Write-Host("Usage:
+Invoke-CadlDataPlaneGenerateSDKPackage.ps1 -service <servicename> -namespace Azure.<group>.<service> -sdkPath <sdkrepoRootPath> -cadlRelativeFolder <relativeCadlProjectFolderPath> [-commit <commitId>] [-repo <specRepo>] [-specRoot <specRepoRootPath>] [-additionalSubDirectories <relativeFolders>]
+
+Options:
+-service  [Required] takes Azure client service directory name in sdk repo. e.g. AnomalyDetector
+-namespace  [Required] takes the namespace of the Azure client library.
+-sdkPath  [Required] takes the address of the root directory of sdk repo. e.g. /home/azure-sdk-for-net
+-cadlRelativeFolder [Required] takes the relative path of the cadl project folder in spec repo. e.g. specification/cognitiveservices/AnomalyDetector
```

Comment:
Can we call this `cadlSpecDirectory` and in the explanation mention its relative?